template.html

```diff
@@ -1,11 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
+  >
   <title>Document</title>
 </head>
+
 <body>
   Este es una plantilla html 5
+  <section>
```

Comment:
```suggestion
  <div>
```
creo que esto deberia ser un div